uicache.1

```diff
@@ -28,50 +28,47 @@
 .Os
 .Sh NAME
 .Nm uicache
-.Nd update registered applications
+.Nd Update registered applications
 .Sh SYNOPSIS
 .Nm
 .Op Fl ahlr
 .Op Fl i Ar id
 .Op Fl p Ar path
 .Op Fl u Ar path
 .Sh DESCRIPTION
-The
-.Nm
-updates the applications that show on the homescreen and optionally will respring afterwards.
-.Pp
-The options are as follows:
+Update iOS registered applications and optionally restart SpringBoard.
+.Sh OPTIONS
```

Comment:
```suggestion
The following options are available:
```

Reply:
This is because `.Sh OPTIONS` is not a standard section of manpages, so just to keep conformity I don't think it should be used.

Reply:
This works for me. Maybe consider using another system; must GNU projects have this on their manpages, so it fits fine.

Reply:
Ok.